### PR TITLE
Get block hash const libfunc

### PR DESCRIPTION
**Pull request by toni-calvin**
# TITLE

## Description

Description of the pull request changes and motivation.

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: programs/class_hash_ex.cairo
Comment (azteca1998) [inline programs/class_hash_ex.cairo:1]: Usually we don't create an example for each libfunc, but instead just add tests when appropiate.
Check out some [example tests](https://github.com/lambdaclass/cairo_native/blob/b7f4a775cc5afcfcf7daf101962726342e9c140a/src/libfuncs/felt252.rs#L291-L625) for reference if you add them.
Keep in mind that adding tests for starknet will require you to implement the trait `StarkNetSyscallHandler`.

File: src/libfuncs/stark_net.rs [hunk 1]
```diff
@@ -530,6 +532,39 @@ where
...
+where
+    TType: GenericType,
+    TLibfunc: GenericLibfunc,
+    <TType as GenericType>::Concrete: TypeBuilder<TType, TLibfunc, Error = CoreTypeBuilderError>,
+    <TLibfunc as GenericLibfunc>::Concrete: LibfuncBuilder<TType, TLibfunc, Error = Error>,
+{
+    let value = match info.c.sign() {
+        Sign::Minus => (&info.c).neg().to_biguint().unwrap(),
+        _ => info.c.to_biguint().unwrap(),
+    };
```
[the remark this answers is not in the document]
Comment (toni-calvin) [reply src/libfuncs/stark_net.rs:553]: That code is used to build the contract address, the base address and now the class hash const, should we update the other ones too?

[the remark this answers is not in the document]
Comment (azteca1998) [reply src/libfuncs/stark_net.rs:553]: Oh you're right.
I've just looked into [the CASM implementation](https://github.com/starkware-libs/cairo/blob/main/crates/cairo-lang-sierra-to-casm/src/invocations/misc.rs#L41-L48) and there's no absolute value.

Could you please fix the others too?

[the remark this answers is not in the document]
Comment (toni-calvin) [reply src/libfuncs/stark_net.rs]: done

**Review by edg-l — APPROVED** [review]

**Review by edg-l — APPROVED** [review]

**Pull request merged**